### ISSUE TITLE
HTML tree: Accessibility problems

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2261]
Testing with NVDA on Windows

1. In the collections tree, NVDA reads each row when selected via the keyboard, but it says "not selected".
2. In the items tree, rows are not announced at all when moving between rows. It should read the text of all visible columns, in order.
3. No position ("2 of 2") or level ("level 1") is announced for rows.
4. When a list is selected, it says "table" instead of "tree view".
5. When tabbing into the items tree, a row isn't selected by default, which makes it hard to know what has focus.

These are all regressions from the previous tree.

Also:

6. The item type, colored tags, and non-matching (gray) rows aren't indicated in any way. They should probably be read before or after the title. E.g., "book The Great Gatsby", "non-matching book The Great Gatsby", "book The Great Gatsby tag Class". We'll talk to some people to figure out the best approach here, but getting them into the row would be a good start.

I assume that some of these things can be addressed with ARIA attributes.
